Scripting/Bindings.cs

```diff
@@ -256,11 +444,86 @@ public override void _Process(float Delta)
 					Scripting.ConsoleEngine.Execute($"{Binding.Function}(1)", Scripting.ConsoleScope);
 				}
 			}
+			else if(Binding.Type == TYPE.CONTROLLERAXIS)
+			{
+				int VerticalAxis = 0;
+				int HorizontalAxis = 0;
+				InputEventJoypadMotion StickEvent = null;
+
+				foreach(InputEvent Option in InputMap.GetActionList(Binding.Name)) {
+					if (Option is InputEventJoypadMotion JoyEvent) {
+						StickEvent = JoyEvent;
+					}
+				}
+
+				if (StickEvent.Axis == 0 || StickEvent.Axis == 1)
+				{
+					// We are using Left stick
+					VerticalAxis = 1;
+					HorizontalAxis = 0;
+				}
+				else if (StickEvent.Axis == 2 || StickEvent.Axis == 3)
+				{
+					// We are using Right stick
+					VerticalAxis = 3;
+					HorizontalAxis = 2;
+				}
+				else
+				{
+					Console.ThrowLog("This joystick doesn't exist! ?????????");
```

Comment:
:laughing: This message is hilarious

Also my bad this should be a ThrowPrint I'm a dope sometimes
See https://github.com/ForLoveOfCats/SkyOfSteel/blob/8d862bfb4f14c211a05bfae93cb8859824d32925/Scripting/Bindings.cs#L369